ql.mm

```diff
@@ -10305,84 +9783,82 @@ Kalmbach axioms (soundness proofs) that require WOML
     GKOWEUSVSVOFZFUSVBVOFZFZVEWDWFUSWCVSVOWBUQBUQUHLKOWFWGUSVSVBVOBUQRKOWHUSVBF
     USUQFZUSBFZEVEWGVBUSVBUSUKOUSUQBUSAABPUIABQULWIUTWJBUSUQUMUSBABUNUOHTTTTTTU
     TUPBUPUTVFJST $.
-    $( [3-Jul-05] $) $( [29-Jun-05] $)
 
   $( Equation 4.10 of [MegPav2000] p. 23.  This is the second part of the
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.2e2 $p |- ( ( a ->1 b ) ^ b ) = ( ( a ^ b ) v ( a ' ^ b ) ) $=
     ( u1lemab ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.11 of [MegPav2000] p. 23.  This is the first part of the
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.3le1 $p |- ( a ' ->1 b ) ' =< a ' $=
     ( u1lem9a ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.11 of [MegPav2000] p. 23.  This is the second part of the
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.3le2 $p |- a ' =< ( a ->1 b ) $=
     ( u1lem9b ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.12 of [MegPav2000] p. 23.  (Contributed by Roy F. Longton,
-     3-Jul-05.) $)
+     1-Jul-2005.)  (Revised by Roy F. Longton, 3-Jul-2005.) $)
```

Comment:
The `Contributed` line should be 3-Jul-2005 and there should be no `Revised by`. I realize that the original has both dates but there's no point in having two dates two days apart.